src/components/VmDetails/cards/NicsCard/NicListItem.js

```diff
@@ -41,14 +41,14 @@ const NicListItem = ({ idPrefix, nic, vmStatus, vnicProfileList, isEditing, onEd
       </div>
       <Grid>
         <Row>
-          <Col cols={6} className={style['ip4-container']} id={`${idPrefix}-ipv4`}>
+          <Col style={{ flexBasis: '40%' }} cols={6} className={style['ip4-container']} id={`${idPrefix}-ipv4`}>
             { nic.ipv4.length > 0 &&
               <div>
                 {nic.ipv4.map(ip4 => <div key={`${nic.id}-${ip4}`}>{msg.nicIP4()}: {ip4}</div>)}
               </div>
             }
           </Col>
-          <Col cols={6} className={style['ip6-container']} id={`${idPrefix}-ipv6`}>
+          <Col style={{ flexBasis: '60%', maxWidth: '60%' }} cols={6} className={style['ip6-container']} id={`${idPrefix}-ipv6`}>
```

Comment:
same, `flexBasis: '60%', maxWidth: '60%'`
could go in
`ip6-container`

Reply:
No, because component `Col` set own style parameter, https://github.com/oVirt/ovirt-web-ui/blob/3b70b675f4b89692bc70ec80c5902bd99895c80b/src/components/VmDetails/GridComponents.js#L31-L48

Reply:
Bootstrap grid columns.....
  12 columns = 100%
  6 and 6 = 50% and 50% (this is how it is currently defined)
  5 and 7 = 42% and 58%

So really, all you should need to do is change the `cols` attribute and they'll take up very close to the amount of space you're overriding with the style tag.  I'd rather have you tweak the grid with the columns instead of using the style attribute.

Reply:
Yeah, right, I somehow missed this.

Reply:
> Yeah, right, I somehow missed this.

No worries!  Looks good now.

Reply:
my comment was dumb. Sorry, and thanks @sjd78